### PR TITLE
Support for bulk consumer

### PR DESCRIPTION
**Pull request by bckp**

### CODE REVIEW

File: src/Consumer/Consumer.php [hunk 1]
```diff
@@ -52,62 +53,74 @@ public function getCallback(): callable
...
+				$client->stop();
 
-					default:
-						throw new \InvalidArgumentException(
-							"Unknown return value of consumer [{$this->name}] user callback"
-						);
-				}
+				break;
 
-				if ($maxMessages !== null && ++$this->messages >= $maxMessages) {
```
Comment (paveljanda) [inline src/Consumer/Consumer.php:104]: This line was lost from previous version. Could you please alter the code to preserve current functionality?

If we are not 100% sure what all the previous code does, we don't need to change any Consumer code and only put the new one in BulkConsumer. I will leave that up 2 you

Comment (bckp) [reply src/Consumer/Consumer.php:104]: We have fixed that, it was lost in refactoring code to de-duplicate it.

File: .docs/README.md [hunk 2]
```diff
@@ -167,6 +167,50 @@ final class TestConsumer implements IConsumer
...
+            $return[$tag] = IConsumer::MESSAGE_ACK; // Or ::MESSAGE_NACK || ::MESSAGE_REJECT
+         }
+	    
+		return $return; 
+	}
+
+}
+```
+
+
```
Comment (paveljanda) [inline .docs/README.md:213]: Could you please fix the indentation? Thanks

**Review by paveljanda — CHANGES_REQUESTED** [review]

File: .docs/README.md (hunk 2 above)
Comment (bckp) [reply .docs/README.md:213]: done, spaces instead of tabs, sorry :)

**Review by paveljanda — APPROVED** [review]

**Comment by paveljanda** [comment]
Thanks!!!

**Pull request merged**